### PR TITLE
Changes to address review from editors

### PR DESCRIPTION
**Pull request by nabobalis**

### CODE REVIEW

File: sections/2-code.tex [hunk 1]
```diff
@@ -20,6 +20,7 @@ \subsubsection{Components of the Core Package}
 \label{sssec:components-of-the-core-package}
 
 To search for and download data, \sunpypkg provides the \Fido interface for searching across a variety of data providers (e.g., the Virtual Solar Observatory (VSO)\footnote{\url{https://sdac.virtualsolar.org/cgi/search}}, or the Joint Science Operations Center (JSOC)\footnote{\url{http://jsoc.stanford.edu}}) maintained within the solar community.
+\Fido itself is a collection of classes that define formal API for users as well as the the search attributes and how each data provider interprets those attributes in order to do the actual search internally.
```
Comment (nabobalis) [inline sections/2-code.tex]:
```suggestion
\Fido internally is a collection of classes that define the search API for users including the search attributes and how a data provider interprets those attributes in order to do the final network call to any server.
```

File: sections/2-code.tex [hunk 2]
```diff
@@ -33,6 +34,11 @@ \subsubsection{Components of the Core Package}
 Additionally, by extending the \astropypkg coordinates framework \citep[see Section 3.3 of][for more details]{the_astropy_collaboration_astropy_2018}, \sunpypkg provides definitions of, and transformations between, common solar coordinate systems.
 Coordinates expressed using these frames can be used to represent the positional information of solar features and events. \sunpypkg implements both observer-dependent (e.g., \hpc) and observer-independent (e.g., \hgs) coordinate frames \citep{thompson_coordinate_2006}.
 Each \code{Map} object instance also carries with it the corresponding coordinate frame of that image and the coordinate of the observer as defined by the position of the observatory given in the associated metadata.
+Whereas for \code{TimeSeries}, the metadata is commonly in a very different style, so there is generally little overlap between the metadata of a \code{TimeSeries} object and the metadata of a \code{Map} object.
```
Comment (nabobalis) [inline sections/2-code.tex]:
```suggestion
Whereas for \code{TimeSeries}, the metadata has a very different style as a result that there is no common format for timeseries data.
Therefore in practice there is generally little overlap between the metadata of a \code{TimeSeries} object and the metadata of a \code{Map} object.
```

File: sections/2-code.tex [hunk 3]
```diff
@@ -33,6 +34,11 @@ \subsubsection{Components of the Core Package}
...
+
+As we will cover in \ref{ssec:affiliated-packages}, the \sunpypkg package part of a larger ecosystem of packages that provide additional functionality for solar physics research.
+So one might ask why \sunpypkg is not simply a collection of these packages, why is there a separation of \enquote{core} vs \enquote{non-core} packages?
```
Comment (nabobalis) [inline sections/2-code.tex]:
```suggestion
So one might ask why is the \sunpypkg not simply a collection of these packages, why is there a separation of \enquote{core} vs \enquote{non-core} functionally?
```

File: sections/2-code.tex [hunk 4]
```diff
@@ -33,6 +34,11 @@ \subsubsection{Components of the Core Package}
...
+The answer is that the \sunpypkg package, the core, is to provide a common set of tools that allow a user to do basic solar physics research without having to install anything else.
```
Comment (nabobalis) [inline sections/2-code.tex]:
```suggestion
The \sunpyproj defines the \enquote{core}, as the tools needed to do standard solar physics research.
This means in practice, finding and downloading data, opening the data and plotting the data.
Scientists will analyse data in so many bespoke ways that adding analysis tools without bloating \sunpypkg package is impossible.
Therefore the decision was made to ensure that the \enquote{core} enabled scientists to get to the point where they can apply their own tools on top.
```

File: sections/2-code.tex [hunk 5]
```diff
@@ -33,6 +34,15 @@ \subsubsection{Components of the Core Package}
 Additionally, by extending the \astropypkg coordinates framework \citep[see Section 3.3 of][for more details]{the_astropy_collaboration_astropy_2018}, \sunpypkg provides definitions of, and transformations between, common solar coordinate systems.
 Coordinates expressed using these frames can be used to represent the positional information of solar features and events. \sunpypkg implements both observer-dependent (e.g., \hpc) and observer-independent (e.g., \hgs) coordinate frames \citep{thompson_coordinate_2006}.
 Each \code{Map} object instance also carries with it the corresponding coordinate frame of that image and the coordinate of the observer as defined by the position of the observatory given in the associated metadata.
+Whereas for \code{TimeSeries}, the metadata has a very different style as a result that there is no common format for timeseries data.
+Therefore in practice there is generally little overlap between the metadata of a \code{TimeSeries} object and the metadata of a \code{Map} object.
```
Comment (nabobalis) [inline sections/2-code.tex]: I know they asked this in the review, but I feel this is kind of pointless.

File: sections/2-code.tex [hunk 6]
```diff
@@ -20,6 +20,7 @@ \subsubsection{Components of the Core Package}
 \label{sssec:components-of-the-core-package}
 
 To search for and download data, \sunpypkg provides the \Fido interface for searching across a variety of data providers (e.g., the Virtual Solar Observatory (VSO)\footnote{\url{https://sdac.virtualsolar.org/cgi/search}}, or the Joint Science Operations Center (JSOC)\footnote{\url{http://jsoc.stanford.edu}}) maintained within the solar community.
+\Fido internally is a collection of classes that define the search API for users including the search attributes and how a data provider interprets those attributes in order to do the final network call to any server.
```
Comment (nabobalis) [inline sections/2-code.tex]: This doesn't really help.

File: main.tex [hunk 7]
```diff
@@ -20,6 +20,8 @@
 \newcommand{\astropy}{Astropy\xspace}
 \newcommand{\astropypkg}{\package{astropy}}
 \newcommand{\aiapypkg}{\package{aiapy}}
+\newcommand{\pfsspypkg}{\package{pfsspy}}
+\newcommand{\soarpkg}{\package{sunpy\_soar}}
```
Comment (nabobalis) [inline main.tex:24]: Ah I should have escaped it. Doh

**Comment by nabobalis** [comment]
precommit unhappy:

```
sections/2-code.tex:9:While other packages in the ecosystem may focus on particular analysis techniques or analyzing data from specific instruments, the \sunpypkg ``core'' package is focused on providing general tools for working with solar physics data.
```

**Review by nabobalis — COMMENTED** [review]
Thanks Will :+1:

File: sections/2-code.tex [hunk 8]
```diff
@@ -5,30 +5,34 @@ \subsection{The \sunpypkg core package}
...
+Once a user has downloaded data, the \code{TimeSeries} and \code{Map} objects can be used to load and visualize time series and two-dimensional image data, respectively.
 These objects hold the data alongside the associated metadata in order to perform metadata-aware operations such as concatenation for time series or cropping for image data.
 In the case of \code{Map}, a World Coordinate System \citep[WCS, e.g.,][]{greisen_representations_2002} is also constructed from the associated metadata to enable easy mapping between pixel and world coordinates via \astropypkg.
+\added[id=WTB]{In nearly all cases, solar image data is stored in the FITS format }\citep{wells_fits_1981} \added[id=WTB]{which has an accompanying well-defined metadata standard} \citep{pence_definition_2010}.
+\added[id=WTB]{The accompanying metadata for each \code{Map} object adheres to this standard.
+Solar time series data, however, do not have a standard metadata or file format and may be stored in a variety of file formats, including FITS, netCDF, JSON, or plain text.}
```
Comment (dstansby) [inline sections/2-code.tex]:
```suggestion
Solar time series data, however, do not have a standard metadata or file format and are stored in a variety of file formats, including FITS, netCDF, JSON, or plain text.}
```

File: sections/2-code.tex [hunk 9]
```diff
@@ -5,30 +5,34 @@ \subsection{The \sunpypkg core package}
...
+\added[id=NF]{As such, the metadata associated with each \code{TimeSeries} object is much more sparse compared to \code{Map}, but at minimum will include the time of each observation as well as the associated instrument that made the observation.}
```
Comment (dstansby) [inline sections/2-code.tex]:
```suggestion
\added[id=NF]{As such, the metadata associated with each \code{TimeSeries} object is much more sparse compared to \code{Map}, but at minimum will include the time of each observation as well as some information about the associated instrument that made the observation.}
```

**Comment by nabobalis** [comment]
We can just ignore the pre commit, enquote and added don't seem to play well together.

**Comment by wtbarnes** [comment]
Lol yeah sorry I should've stuck that in a comment. I had to move those to the usual LaTeX quote style because of the track changes markup.

**Pull request merged**